README.md

```diff
@@ -86,6 +88,29 @@ Example wrapper cookbooks:
 - [mod](documentation/resource_apache2_mod.md)
 - [module](documentation/resource_apache2_module.md)
 
+## Contributors
+
+This project exists thanks to all the people who [contribute.](https://opencollective.com/sous-chefs/contributors.svg?width=890&button=false)
+
+### Backers
+
+Thank you to all our backers! 🙏 [Become a backer](https://opencollective.com/sous-chefs#backer)
+![https://opencollective.com/sous-chefs#backers](https://opencollective.com/sous-chefs/backers.svg?width=890)
```

Comment:
This could do with being on a second line here. 

I think *think* the layout looks better